### PR TITLE
Fixes, Removed else if statements and added dynamic command handling 

### LINKED ISSUE #61: Add command handlers
We currently use `else.. if` chains for various commands implemented in the bot. It is fine for the very few commands we have but as we scale to add more commands and more features it will cause a lot of unorganized code and make it harder to debug. So we should implement a handler for that.

The discord.js guide has an amazing explanation of how to implement it. 
https://discordjs.guide/command-handling/

**If it is the first time that you contribute to the bot, follow these steps:**

Write a comment in this issue thread to let other possible contributors know that you are working on this bug. For eg : `Hey all, I would like to work on this issue.`, follow [Contributing to the bot 🔥✨](https://github.com/osdc/bots/blob/master/discord-bot/README.md) and feel free to ask anything related to this issue ✌️

### PR DESCRIPTION
**Pull request by reapedjuggler**
Fixes #61, Implemented the code using dynamic command handling instead of else if statements, All the commands are now .js files under the command directory with their respective name.

### CODE REVIEW

**Comment by reapedjuggler** [comment]
Here is a screenshot of the working.
Screenshot_20201111-215458__01__01

File: discord-bot/commands/irc.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const Discord = require('discord.js')
+
+module.exports = {
+    name: '!irc',
+    description: 'irc Link',
```
Comment (daemon1024) [inline discord-bot/commands/irc.js]:
```suggestion
    description: 'IRC Link',
```

**Review by daemon1024 — CHANGES_REQUESTED** [review]
Small nit, Rest LGTM :smile: Also can you please resolve the merge conflicts? Thanks. Sorry for the delayed review :)

**Comment by reapedjuggler** [comment]
No worries, done with the changes :)

**Review by daemon1024 — APPROVED** [review]

**Pull request merged**